app/src/main/java/com/babylon/wallet/android/domain/usecases/transaction/SendClaimRequestUseCase.kt

```diff
@@ -42,7 +42,7 @@ class SendClaimRequestUseCase @Inject constructor(
                 }
             )
         }.mapCatching { manifest ->
-            TransactionManifestData.from(manifest).prepareInternalTransactionRequest()
+            UnvalidatedManifestData.from(manifest).prepareInternalTransactionRequest()
```

Comment:
I see that `TransactionManifestData` is now "renamed" to `UnvalidatedManifestData`. This makes sense in cases were we haven't validated the instructions yet. But in cases like this the manifest is ready and validated.  It looks like `TransactionToReviewData` is more relevant here. Maybe it would be better to rename it?

Reply:
You're right. I'll think about it and try to make the change as part of my next PR.